docs/tau-updates.md

```diff
@@ -0,0 +1,122 @@
+# Targetability Updates (TAU)
+
+In cases where fflogs sends insufficient information to determine the correct information and state of a boss (invulnerability window, ephemeral enemies, etc), there is a mechanism to override this on a per-fight basis under `src/parser/bosses`.
+
+# Setup
+
+## Scaffolding
+
+At current, you'll need to provide the following scaffolding / directory structure:
+
+```
+src/parser/bosses/
+	${IDENTIFIER}/
+		index.ts
+		modules/
+			Invulnerability.ts
+			index.ts
+```
+
+Where `${IDENTIFIER}` is a meaningful identifier for a boss. It's important to note that in the case of savage fights, the boss id is the same between normal and savage, so there's no need to make normal and savage scaffolding for a fights.
```

Comment:
I don't think this is right, unless Kihra fixed shit, E9 and E9S seem to have different IDs.

Reply:
Seems fflogs IDs rather than game, disregard.